cli/cmd/version.go

```diff
@@ -11,11 +11,20 @@ import (
 	"github.com/microsoft/retina/internal/buildinfo"
 )
 
+// This variable is used by the "version" command and is set during build.
+// Default to a safe value if not set.
+var defaultVersion = "v0.0.5"
+
 var version = &cobra.Command{
 	Use:   "version",
 	Short: "Show version",
 	Run: func(*cobra.Command, []string) {
-		fmt.Println(buildinfo.Version)
+		if buildinfo.Version != "" {
+			fmt.Println(buildinfo.Version)
+		} else {
+			fmt.Println("buildinfo.Version is not set successfully, showing default version:")
+			fmt.Println(defaultVersion)
```

Comment:
I think you can probably get a more useful "default" value out of https://pkg.go.dev/runtime/debug#BuildInfo . There's a property in there for the `vcs.revision` which should be the commit the binary was built with. It won't be the tag, but the commit is plenty useful (and would be handy for development builds).